### DISCUSSION TITLE
Anfrage wegen REETZ_IMAGE_NAME_PREFIX

### DISCUSSION THREAD

**Discussion by freetz-ng-mod** [Freetz-NG/freetz-ng#212] [Unerfüllte Wünsche]
Hi fda,

ich habe mal eine frage. Gibt es eine Möglichkeit FREETZ_IMAGE_NAME_PREFIX an eine andere position zu setzen als ganz vorne. ZB. Vor .image

**Comment by fda77** [comment]
Dann wäre es doch kein PRÄfix mehr! https://de.wikipedia.org/wiki/Pr%C3%A4fix

**Comment by freetz-ng-mod** [comment]
OK !!!!

Danke für den guten WiKI link.

Ich wusste was es heißt. Gibt es eine Chance in freetz-ng ein end name zu bekommen. Da es hier nicht um den Namen geht sondern mehr um Infos was in freetz drin ist

**Comment by fda77** [comment]
Wenn man eine neue Option erstellt muss man der schon einen Namen geben. Was schlägst du denn vor? Endfix find ich nicht so gut. Idefix hat auch schon ne andere Bedeutung
Da das ganze aber ne triviale Angelegenheit ist könntest du doch einen Patch/Pullrequest machen

**Comment by freetz-ng-mod** [comment]
Es geht um openssl ob nun 102 drin ist oder 1.1.1 oder 3.0.0.  Dann baue den Namen ein.

So kannst du dein perFix weiter nutzen

**Reply by fda77** [reply]
Bau doch dein oscam einfach als normales Package in Freetz ein, dann wird die Binary immer passend zur aktuellen Auswahl gebaut

**Comment by freetz-ng-mod** [comment]
Es geht hier nicht um OSCAM.
Aber die Info ob es 1.0.2 oder 1.1.1x oder sogar 3.0 ist wer auch nicht schlecht..

**Reply by fda77** [reply]
Wen soll das interessieren? Du weisst doch was du bei deinem Image ausgewählt hast. Und alle Packages die ausgewählt wurden nehmen automatisch die Ausgewählte

**Reply by Conan179** [reply]
Mich inntresiert es auch, für wen, interessant seihen soll, welche openssl version installiert ist.

**Comment by freetz-ng-mod** [comment]
Du hast keine anung was das für eine aus wirkun hate das du openssl 1.1.1 eingebaut hast.
Es gibt so viele, Programme die auf openssl aufbauen. und dein openssl 1.0.2 zb wurde als 1.0.0 eingebaut. Nun muss man jedes image sich anschauen ob 1.0.2 111 oder sorge 3 drin ist)

**Reply by fda77** [reply]
Welches Image muss man sich anschauen? Ich baue ein Image, flashes es und dann wirds gelöscht
Wenn du lieber veraltete und nicht mehr gewartete openssl Versionen nutzen möchtest kannst du das ja tun

**Comment by freetz-ng-mod** [comment]
Lassen wir das. Dann werde ich es in mein freetz ändern.



⁣BlueMail for Android herunterladen ​

**Reply by fda77** [reply]
> Du hast keine anung was das für eine aus wirkun hate das du openssl 1.1.1 eingebaut hast.

Worin besteht dein Problem die Version auszuwählen die dir am besten gefällt?
